internal/testutil/gpgkeys.go

```diff
@@ -0,0 +1,198 @@
+package testutil
+
+import (
+	"log"
+
+	"golang.org/x/crypto/openpgp/packet"
+
+	"github.com/canonical/chisel/internal/setup"
+)
+
+type Key struct {
+	ID                string
+	ArmoredPublicKey  string
+	ArmoredPrivateKey string
+	PublicKey         *packet.PublicKey
+	PrivateKey        *packet.PrivateKey
+}
+
+var gpgKeys = map[string]*Key{
+	"ubuntu-archive-key-2018": {
+		ID:               "871920D1991BC93C",
+		ArmoredPublicKey: ubuntuArchiveSignKey2018,
+	},
+	"test-key": {
```

Comment:
[Nit]: Can we remove this to `test-key-1`, right now it looks like the `test-key` is somehow special.

Reply:
Changed in https://github.com/letFunny/chisel/pull/1/commits/f13b23f0fdc693cb802e2fc47ab041fe0e227bec.